examples/with-tailwindcss/package.json

```
@@ -7,15 +7,14 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "^9.1.8-canary.11",
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0"
+    "next": "^9.2.0",
```

```suggestion
    "next": "latest",
```
We can set this to latest now that built-in CSS support has landed on stable